### ISSUE TITLE
quesiton; multi server network?

### ISSUE THREAD

**Issue by Merith-TK** [caunt/BungeeForge#17]
My server network has a Purpur, Fabric, (1.20.4) and soon to be if this mod works out, forge server (1.12.2)

Will this work as a drop in addon to my system?

**Comment by p0t4t0sandwich** [comment]
It should work with Forge 1.12.2 yeah.
For anything Forge 1.13+ you need to use Velocity with Ambassador

**Comment by mckzstudios** [comment]
Maybe they don't want to use shitty Velocity and want to use a real system like BungeeCord which has actual backing and a full team behind it.

**Comment by Merith-TK** [comment]
Was meaning as <1.12.2 AND >1.13 servers side by side

**Comment by p0t4t0sandwich** [comment]
Yep, though only if you use Velocity, Ambassador, and Legacy/BungeeGuard forwarding.

Edit: if you want Forge 1.13+ you need Ambassador, otherwise as long as you're using BungeeCord/Legacy forwarding it should work fine with Forge 1.12.2 and below
